umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md

```diff
@@ -146,7 +162,9 @@ http://<instance-ip-address>:1880/nodered
 #### Access Grafana
 
 Grafana, an open-source analytics and monitoring solution, is used by UMH for
-dashboard displays. Retrieve the credentials with these commands:
+dashboard displays.
+
+After logging in as the root user, retrieve the credentials with these commands:
 
 ```bash
 kubectl get secret grafana-secret -n united-manufacturing-hub -o jsonpath="{.data.adminuser}" | base64 --decode; echo
```

Comment:
add the --kubeconfig directly in there. don't use the path, people will forget about it